### PR TITLE
Do not merge yet: spatial data

### PR DESCRIPTION
**Pull request by nevrome**
PR to keep track of the changes

### CODE REVIEW

**Comment by dakni** [comment]
I would not yet merge "coordinate_precision.R" since we are still in the phase of writing/finishing/testing the functions; 

the file from the playground should be deleted before merging

the file "determine_country_by_coordinate.R" is ok to be merged, though it is also still undertested.

**Comment by nevrome** [comment]
As the title of the PR already says: **Do not merge yet**

I just really like the PR interface of github to track changes. :-)

**Comment by dakni** [comment]
ahja... ;) 

in this case: how is it possible to indicate what to be merged and what not? is it possible to review an entire file or am can only review single lines?

and...what do you mean by tracking changes? where is the difference to a simple commit log!?

**Comment by nevrome** [comment]
I'm not aware of a way to exclude certain files or commits from a PR on github. You would need to split that up into different branches and PRs. 
The review can happen line wise or concerning the complete PR. No review on file level (but why would you need that if you can work on every line?).
The commit log is fine. In this case I just wanted to see all the new stuff in direct relation to the master branch. The automatic checks/tests also already consider the master branch here.

File: R/finalize_country_name.R [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  stop("x is not an object of class c14_date_list")
+}
+
+#' @rdname finalize_country_name
+#' @export
+finalize_country_name.c14_date_list <- function(x) {
+  
+  # check if the columns country_coord and country_thes are present
+  necessary_vars <- c("country_coord", "country_thes")
+  x %>% check_if_columns_are_present(necessary_vars)
```
Comment (dakni) [inline R/finalize_country_name.R]: there is something missing here; where does the pipe lead to?

Comment (kschmuetz) [reply R/finalize_country_name.R]: I was hoping, that if the check was positive it wouldn´t have to lead anywhere and just move on and if the test is negative the function would be stopped by the check

File: R/finalize_country_name.R [hunk 2]
```diff
@@ -27,33 +27,33 @@ finalize_country_name.default <- function(x) {
...
-    
+
     # pick the most likely result hierarchically from country_coord, country_thes
     # and the original input data
     x %<>%
       dplyr::mutate(
         if (country_coord != NA) {
           country_final = country_coord
-        } else if (country_thes ! NA) {
+        } else if (country_thes != NA) {
```
Comment (dakni) [inline R/finalize_country_name.R]: you do not need `else if`

either use `else`or `if`

**Comment by nevrome** [comment]
Is your work ready for a review?

**Comment by dakni** [comment]
we did not yet tested it on other data than adrac. I first blink indicated an error with other data sources. We will face these issues during the next ISAAK meeting this Thursday.

**Comment by dakni** [comment]
TODO: `rwoldxtra` and `sf` are not automatically loaded when the `determine_country_by_coordinate` function is called

**Pull request merged**